mitmproxy/addonmanager.py

```diff
@@ -88,7 +91,7 @@ def add_option(
             if same_signature:
                 return
             else:
-                ctx.log.warn("Over-riding existing option %s" % name)
+                logger.warning("Over-riding existing option %s" % name)
```

Comment:
call-out for the future PRs: let's move towards f-strings when we touch such a line.